src/utils/objects.test.js

```diff
@@ -51,12 +58,48 @@ describe('objects util', () => {
         geometries: [
           {
             type: 'ShapeGeometry',
-            shapes: [Array],
+            shapes: [],
             curveSegments: 12,
           },
         ],
         uuids: [undefined, undefined],
       },
     })
   })
+
+  it('filterObject', () => {
+    const testObj = deepCloneObject(templateObj)
+    const filteredObject = filterObject(testObj, (val, key) => {
+      // eslint-disable-next-line no-magic-numbers
+      if (val === 3 && key === 'uuid') {
```

Comment:
I don't understand this filter condition.

The object filter methods were added to filter out uuids, but the expected value below leaves them in?

Reply:
Sometimes, parameters behind "?"  are removed before page is loaded, so can't see place mark occasionally.
So, I used hash parameters to not lost place mark feature info.
Test URL is https://deploy-preview-657--bldrs-share.netlify.app/share/v/gh/ronplaymeta/bldrs-ai-Share/main/public/index.ifc#f:placemark

And above filter condition returns a new object that only {'uuid': 3} pairs are removed.